annif/transformer/transformer.py

```diff
@@ -0,0 +1,52 @@
+"""Common functionality for input transforming."""
+
+from annif.corpus import TransformingDocumentCorpus
+from annif.exception import ConfigurationException
+
+
+class IdentityTransformer():
+    """"""""  # TODO
+
+    name = 'pass_through'
+
+    def __init__(self, project):
+        self.project = project
+
+    def transform_text(self, text):
+        """"""  # TODO
+        return text
+
+    def transform_corpus(self, corpus):
+        """"""  # TODO
+        return TransformingDocumentCorpus(corpus, self.transform_text)
+
+
+class Transformer():
```

Comment:
I think it's a little confusing that this `Transformer` class is basically unrelated to the actual transformer classes - it's a container for them but not itself a transformer, at least not on the same level. Based on naming I would expect this to be a superclass of all transformers. I suggest renaming it to `TransformerChain` or something along those lines.